### DISCUSSION TITLE
Cannot execute Vorta.app on Intel Mac

### DISCUSSION THREAD

**Discussion by stefanbuehler** [borgbase/vorta#2127] [FAQ] [answered]
After in-app update, I could not start the Vorta app on my Intel Mac Mini (2018), running the latest OS (Sequoia 15.1). I deleted it and installed fresh via homebrew, following the homepage instructions. Installation went through, but app is again not executable. Trying to run it gives error message that it is not supported on this Mac. Something related to the Intel CPU? Does anyone know how to fix this?

**Comment by m3nu** [comment, marked as answer]
There are now 2 macOS builds for Intel and Arm. Just grab the one that says Intel. I don't know how to make a universal build or ship both via Appcast.

https://github.com/borgbase/vorta/releases/tag/v0.10.0

**Comment by stefanbuehler** [comment]
Thanks!

**Comment by m3nu** [comment]
BTW: Homebrew has a way of linking a separate arm and intel version. They just made this change for Vorta few hours ago. So installing with Homebrew should work from here on.
